This patch originated in-house, to update the Solaris native DB type for
aliases from "dbm" (which was correct for 2.6 and earlier) to "hash" (which
has we have used since Solaris 7).  Upstream contribution is planned.

--- postfix-3.2.2/src/util/sys_defs.h.~1~	2017-02-01 17:31:15.000000000 +0000
+++ postfix-3.2.2/src/util/sys_defs.h	2017-06-15 12:37:58.393559280 +0000
@@ -426,7 +426,7 @@
 #define DEF_MAILBOX_LOCK "fcntl, dotlock"
 #define HAS_FSYNC
 #define HAS_DBM
-#define NATIVE_DB_TYPE	"dbm"
+#define NATIVE_DB_TYPE	"hash"
 #define ALIAS_DB_MAP	DEF_DB_TYPE ":/etc/mail/aliases"
 #ifndef NO_NIS
 #define HAS_NIS
